pkgs/development/compilers/openjdk/openjfx/11.nix

```diff
@@ -1,4 +1,4 @@
-{ stdenv, lib, fetchFromGitHub, writeText, gradle_7, pkg-config, perl, cmake
+{ stdenv, lib, fetchFromGitHub, fetchurl, writeText, gradle_7, pkg-config, perl, cmake
 , gperf, gtk2, gtk3, libXtst, libXxf86vm, glib, alsa-lib, ffmpeg_4-headless, python3, ruby, icu68
```

Comment:
It seems like `icu68` here is not needed at all (still builds without it). And I do not believe it ever was.
It was introduced at the same time when `COMPILE_WEBKIT` was set to `false` in https://github.com/NixOS/nixpkgs/commit/9e5e9c57bb8958004134806eef8e31aed9375ea6
But only module javafx.web uses it, as seen in [this PR](https://github.com/openjdk/jfx17u/commit/04b32321bd22f2e6fa5373a53f67b116bd570a53#diff-a2b6b5de344d88dff056db60630fe08f00118c907ac1c07537941973ae19187e).
But I do not want to remove it in this PR to not cause unnecessary rebuilds.